### PR TITLE
samples: add ADXL362 sensor sample application

### PR DESCRIPTION
**Pull request by bwitherspoon**
Add a sample application that demonstrates how to use the ADXL362 with
data ready and threshold triggers.

This depends on #15913 to work.

### CODE REVIEW

**Comment by bwitherspoon** [comment]
Tested on cc1352r1_launchxl board with SPI support from #15741 and a TI BOOSTXL-ULPSENSE shield not upstream yet.

**Review by dbkinder — APPROVED** [review]
doc looks OK, thanks!

File: samples/sensor/adxl362/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (MaureenHelm) [inline samples/sensor/adxl362/CMakeLists.txt]: Please add a license/copyright header

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: samples/sensor/adxl362/CMakeLists.txt (hunk 1 above)
Comment (bwitherspoon) [reply samples/sensor/adxl362/CMakeLists.txt]: Added

**Review by MaureenHelm — APPROVED** [review]

**Comment by galak** [comment]
rebased on master to re-kick CI.

**Pull request merged**